ts/index.ts

```diff
@@ -84,10 +84,22 @@ interface GossipOptions extends GossipInputOptions {
   scoreThresholds: PeerScoreThresholds
 }
 
+const ACCEPT_REQUEST_WHITE_LIST_THRESHOLD_SCORE = 0
+const ACCEPT_REQUEST_WHITE_LIST_MAX_MESSAGES = 128
+const ACCEPT_REQUEST_WHITE_LIST_DURATION_MS = 1000
+
+interface AcceptRequestWhiteListEntry {
+  /** max number of messages accepted after a score is calculated */
+  messagesAccepted: number
+  /** have to recompute score after this time */
+  whitelistedTill: number
+}
+
 class Gossipsub extends Pubsub {
   peers: Map<string, PeerStreams>
   direct: Set<string>
   seenCache: SimpleTimeCache
+  acceptRequestsWhitelist: Map<string, AcceptRequestWhiteListEntry>
```

Comment:
We can call this the `acceptFromWhitelist`. (and replace "request" elsewhere too)

We should add a comment here to explain the purpose, too.